docs/remote.rst

```diff
@@ -35,16 +37,17 @@ the new rootfs. It they don't then retry from the beginning.
 
 .. _GitHub: https://github.com/PandABlocks/PandABlocks-rootfs/releases
 
-Updating a zpkg
----------------
+Updating zpkg packages
+----------------------
 
-Download a new zpkg files from the appropriate GitHub repositories, then::
+Download new zpkg files from the appropriate GitHub repositories (these are shown at <PandA-URL>/admin/packages/list), then::
 
     $ scp *.zpg root@my_panda_ip:/tmp
     $ ssh root@my_panda_ip
+    # zpkg remove <packages>  # Package names can be found with `zpkg list`
```

Comment:
You shouldn't need to remove packages, any packages installed with the same name automatically removes the old version